packages/grafana-ui/src/components/ContextMenu/WithContextMenu.tsx

```diff
@@ -6,9 +6,11 @@ interface WithContextMenuProps {
   children: (props: { openMenu: React.MouseEventHandler<HTMLElement> }) => JSX.Element;
   /** A function that returns an array of menu items */
   renderMenuItems: () => React.ReactNode;
+  /** Enable focus on first Menu element on openning the menu */
+  focusOnOpen?: boolean;
 }
 
-export const WithContextMenu: React.FC<WithContextMenuProps> = ({ children, renderMenuItems }) => {
+export const WithContextMenu: React.FC<WithContextMenuProps> = ({ children, renderMenuItems, focusOnOpen = true }) => {
```

Comment:
Should we default this optional prop to be `true`?

Reply:
This matches current behavior